building/build-acis/pullcheck/build.sh

```diff
@@ -5,7 +5,6 @@ source ../common/container-build-helpers.sh
 
 VERSION="0.1.0"
 
-DEBVER="stretch.20180111T215606Z"
```

Comment:
What does this change mean?

Reply:
The variable was unused and should not have been copy-pasted to here.